src/main/java/org/folio/circulation/domain/OverdueFineCalculatorService.java

```diff
@@ -278,11 +297,13 @@ private boolean shouldCreateFine(OverdueFinePolicy overdueFinePolicy) {
     private final FeeFineRepository feeFineRepository;
     private final UserRepository userRepository;
     private final FeeFineActionRepository feeFineActionRepository;
+    private final LostItemPolicyRepository lostItemPolicyRepository;
 
     Repos(OverdueFinePolicyRepository overdueFinePolicyRepository,
       AccountRepository accountRepository, ItemRepository itemRepository,
       FeeFineOwnerRepository feeFineOwnerRepository, FeeFineRepository feeFineRepository,
-      UserRepository userRepository, FeeFineActionRepository feeFineActionRepository) {
+      UserRepository userRepository, FeeFineActionRepository feeFineActionRepository,
+      LostItemPolicyRepository lostItemPolicyRepository) {
```

Comment:
I think circulation is in ever increasing need for an anti-corruption layer for fees and fines, that is out of scope of this change, which is already large